index.html

```diff
@@ -2788,6 +2788,23 @@ <h1>DID Methods</h1>
               <a href="https://gitlab.com/proximax-enterprise/siriusid/sirius-id-specs/-/blob/master/docs/did-method-spec.md">ProximaX SiriusID DID Method</a>
           </td>
         </tr>
+      <tr>
+        <td>
+          did:dock:
+        </td>
+        <td>
+          PROVISIONAL
+        </td>
+        <td>
+            Dock
+        </td>
+        <td>
+            [Dock.io](https://www.dock.io)
```

Comment:
`<a href` should be used instead of markdown.

Reply:
Sad that this got through, thanks for the catch @pleerock :) -- @lovesh can you submit another PR to fix this, please?

Reply:
https://github.com/w3c/did-spec-registries/pull/75

I merged this so I fixed it.